mailpoet/lib/AdminPages/PageRenderer.php

```diff
@@ -136,7 +136,10 @@ public function displayPage($template, array $data = []) {
     if ($this->subscribersFeature->isSubscribersCountEnoughForCache($subscriberCount)) {
       $subscribersCacheCreatedAt = $this->transientCache->getOldestCreatedAt(TransientCache::SUBSCRIBERS_STATISTICS_COUNT_KEY) ?: Carbon::now();
     }
-
+    $hideAutomations =
+      !$this->wp->applyFilters('mailpoet_show_automations', false) &&
+      $this->servicesChecker->isBundledSubscription() &&
+      $this->wp->isPluginActive('automatewoo/automatewoo.php');
```

Comment:
I would suggest to make the result of `mailpoet_show_automations` the definitive answer on whether you want to hide or show the automations. 

In the current implementation, the filter is just an additional piece of information on whether to show or hide. I think a better use of the filter would be to return the definitive piece of information on whether to show or hide.

Something along this lines:
```suggestion
    $hideAutomations = $this->wp->applyFilters('mailpoet_show_automations', 
      $this->servicesChecker->isBundledSubscription() &&
      $this->wp->isPluginActive('automatewoo/automatewoo.php');
```

Reply:
You are right. I have changed the code to use the filter as the definite answer.
I changed the condition a bit from what you proposed.